catalog-info.yaml

```diff
@@ -0,0 +1,55 @@
+# Declare a Backstage Component that represents rally-tracks
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: rally-tracks
+  description: This repository contains the default track specifications for the Elasticsearch benchmarking tool Rally.
+  annotations:
+    backstage.io/source-location: url:https://github.com/elastic/rally-tracks
+    github.com/project-slug: elastic/rally-tracks
+    github.com/team-slug: elastic/es-perf
+    buildkite.com/project-slug: elastic/rally-tracks
+  tags:
+    - elasticsearch
+    - benchmark
+    - python
+    - performance
+spec:
+  type: application
+  owner: group:es-perf
+  lifecycle: production
+  dependsOn: 
+    - "resource:rally-tracks-it"
+
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+---
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: rally-tracks-it
+  description: Run tests
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/rally-tracks-it
+
+spec:
+  type: buildkite-pipeline
+  owner: group:es-perf
+  system: buildkite
+
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: IT
```

Comment:
The name will show up here: https://buildkite.com/elastic?team=es-perf so it would be nice to be consistent. Maybe "Rally Tracks - IT". That way the URL slug will be "rally-tracks-it" as already declared in the links section.

Reply:
thanks - renamed